web_responsive/static/src/js/web_responsive.js

```diff
@@ -264,6 +265,14 @@ odoo.define('web_responsive', function (require) {
                 },
             });
         },
+
+        // Load view indicator
+        _onAppsMenuItemClicked: function (ev) {
+            ev.preventDefault();
+            ev.stopPropagation();
```

Comment:
Isn't this done in `this._super()`?

Reply:
no, the click propagates and causes the menu to close